legacy/components/anemone/test/dummy/config/puma.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
```

Comment:
I'd prefer not to have this here. Are you guys using it? And, if so, how?

Reply:
I'm not using it, or actively using/modifying any of the ruby files.